service/src/query_processor.rs

```diff
@@ -71,20 +71,25 @@ impl ToString for SubgraphDeploymentID {
         self.value.to_string()
     }
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Signature {
     v: i64,
     r: String,
     s: String,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[allow(non_snake_case)] // Need exact field names for JSON response
 pub struct QueryResult {
-    graphql_response: String,
-    attestation: Option<Signature>,
+    pub graphQLResponse: String,
+    pub attestation: Option<Signature>,
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[allow(non_snake_case)] // Need exact field names for JSON response
 pub struct UnattestedQueryResult {
-    pub graphql_response: String,
+    pub graphQLResponse: String,
```

Comment:
Perhaps we can use serde alias here to match JSON response?

Reply:
Oh cool didn't know about that solution. I know `strum` can do that too, I used it in the TAP Aggregator service.

Reply:
Fix in [0c02b88](https://github.com/graphops/indexer-service-rs/pull/21/commits/0c02b88971474da1147ffd1ca8565fefbe2c9378)